pages/common/doggo.md

```diff
@@ -0,0 +1,25 @@
+# doggo
+
+> Command-line DNS client for Humans.
```

Comment:
```suggestion
> DNS client for Humans.
```
We avoid using `command-line` in commands description. https://github.com/tldr-pages/tldr/blob/main/contributing-guides%2Fstyle-guide.md#wording

Reply:
Cheers.
Will update and submit...

Seems I tldr'd the contributing guide.